wandb: Tracking run with wandb version 0.14.2
wandb: W&B syncing is set to `offline` in this directory.  
wandb: Run `wandb online` or set WANDB_MODE=online to enable cloud syncing.
Using cpu device
Traceback (most recent call last):
  File "/nas/ucb/mason/ethically-compliant-rl/tests/PPO/train_ppo.py", line 156, in <module>
    train()
  File "/nas/ucb/mason/ethically-compliant-rl/.venv/lib/python3.9/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/nas/ucb/mason/ethically-compliant-rl/tests/PPO/train_ppo.py", line 112, in train
    agent = PPO("MlpPolicy",
  File "/nas/ucb/mason/ethically-compliant-rl/stable_baselines3/stable_baselines3/ppo/ppo.py", line 167, in __init__
    self._setup_model()
  File "/nas/ucb/mason/ethically-compliant-rl/stable_baselines3/stable_baselines3/ppo/ppo.py", line 170, in _setup_model
    super()._setup_model()
  File "/nas/ucb/mason/ethically-compliant-rl/stable_baselines3/stable_baselines3/common/on_policy_algorithm.py", line 133, in _setup_model
    self.policy = self.policy_class(  # type: ignore[assignment]
  File "/nas/ucb/mason/ethically-compliant-rl/stable_baselines3/stable_baselines3/common/policies.py", line 521, in __init__
    self._build(lr_schedule)
  File "/nas/ucb/mason/ethically-compliant-rl/stable_baselines3/stable_baselines3/common/policies.py", line 595, in _build
    self.value_net = nn.Linear(self.mlp_extractor.latent_dim_vf, 1, inc_bias_rng_pos=self.mlp_extractor.latent_dim_vf) # should be (inc_bias_rng_pos=self.mlp_extractor.latent_dim_vf-1)*n_costs
TypeError: __init__() got an unexpected keyword argument 'inc_bias_rng_pos'
wandb: Waiting for W&B process to finish... (failed 1).
wandb: You can sync this run to the cloud by running:
wandb: wandb sync /nas/ucb/mason/ethically-compliant-rl/wandb/offline-run-20240205_032209-srb8gexa
wandb: Find logs at: ./wandb/offline-run-20240205_032209-srb8gexa/logs
srun: error: sac.ist.berkeley.edu: task 0: Exited with exit code 1
